wiki/en/en-Getting-Started.md

```diff
@@ -15,13 +15,13 @@ permalink: "/wiki/Getting-Started"
 
 ## Maximise quality, minimise delay
 
-### 1. Use an audio interface/external microphone, not your internal sound card
+### 1. Minimize internet traffic and close all programs on your PC
```

Comment:
Jamulus does work without an audio interface although it's not recommended. I would nevertheless stress internet connectivity more. But that's debatable.

Reply:
The term "Minimize internet traffic" is a little obscure I think. Perhaps better as "Close all programs that might also be using the Internet" (BTW I also capitalise the word "Internet" as a mark of respect - like "Pacific" or "Parliament" but I'm not sure how consistent I've been with that).

Reply:
Actually, I better not comment on this using Github as I can't see the formatting or overall context of this so ignore the above for now.

Reply:
Ok. By the way: happy birthday (I know I'm late). 

I will look at your comments tomorrow.

Reply:
> Close all programs that might also be using the Internet

Closing programs also has the nice side effect on windows and macOS that jamulus/jamulus audio works since the sound card is then accessible for jamulus. On macOS Jamulus didn't open because I was doing a screen recording. After I stopped it Jamulus opened.

Reply:
> You can start a new review and see every change there. 

OK I think I've worked it out now.